pages.pt_BR/common/clamdscan.md

```diff
@@ -0,0 +1,36 @@
+# clamdscan
+
+> Faz uma varredura em busca de vírus usando o ClamAV Daemon.
+> Mais informações: <https://docs.clamav.net/manual/Usage/Scanning.html#clamdscan>.
+
+- Faz uma varredura em um arquivo ou diretório por vulnerabilidades:
+
+`clamdscan {{caminho/para/arquivo_ou_diretório}}`
+
+- Faz uma varredura nos dados da `stdin` (entrada padrão):
+
+`{{comando}} | clamdscan -`
+
+- Faz uma varredura no diretório atual e lista apenas os arquivos infectados:
+
+`clamdscan --infected`
+
+- Gera um relatório da varredura para um arquivo de registro:
+
+`clamdscan --log {{caminho/para/arquivo_de_registro}}`
+
+- Gera um relatório da varredura para um arquivo de registro:
```

Comment:
```suggestion
- Move arquivos infectados para um diretório específico:
```

Reply:
WTH did I do there? Thanks for the attention!